pyani_plus/db_orm.py

```diff
@@ -245,6 +246,53 @@ def __repr__(self) -> str:
         )
 
 
+class Run(Base):
+    """Describes a single pyANI-plus run.
+
+    Each run is a set of all-vs-all comparisons between a set of FASTA
+    files (genome table rows), for a specific ANI algorithm or method
+    (represented as a row in the configuration table),
+    recorded as a set of comparison table rows.
+
+    Thus one Run is linked to one Configuration row, many Genome rows,
+    and many Comparison rows, where each Comparison is linked to two
+    Genome rows (query and reference).
+
+    In order to generate reports and plots quickly, the Run entry also
+    caches dataframes of the information in the linked Comparison entries
+    (as JSON-encoded Pandas dataframes).
```

Comment:
I'm agnostic about whether we use Pandas or something more lightweight, eventually. We don't need all the bells and whistles of Pandas to work with the data - but we're OK with it for now. I'm not sure how well it will scale in future, though.

Reply:
OK in the short term I think (1000s of genomes), followup PR implements this.